ple/games/donkeykong/Button.py

```diff
@@ -0,0 +1,30 @@
+__author__ = 'Batchu Vishal'
+
+'''
+This class defines all our buttons
+A button has an image and a position associated with it
+'''
+
+
+class Button:
```

Comment:
Is `Button` being used in the code? I believe this was only used for the menu. If its not can you remove it?